### PR TITLE
changed to HauserGroup

### LINKED ISSUE #6: Change to `HauserGroup` in `setup.py`
https://github.com/HauserGroup/gogoGPCR/blob/23b33aca796895c29572e983cdc1e956f127dd05/setup.py#L43

### PR DESCRIPTION
**Pull request by Crixcrs**
close #6

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
     #     ],
     # },
     project_urls={
-        "Bug Reports": "https://github.com/jsture/gogoGPCR/issues",
+        "Bug Reports": "https://github.com/HauserGroup/gogoGPCR/issues",
```
Comment (jsture) [inline setup.py:43]: This line looks very good

File: setup.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
...
         "DNAnexus environment": (
```
Comment (jsture) [inline setup.py:44]: We should also fix this in the future

File: setup.py [hunk 3]
```diff
@@ -40,7 +40,7 @@
...
             "https://ukbiobank.dnanexus.com/panx/tools/workers/jupyterLab"
```
Comment (jsture) [inline setup.py:45]: This URL with a comment

File: setup.py (hunk 2 above)
Comment (petergerlach1) [inline setup.py:44]: Very nice code 🗡️

**Review by jsture — APPROVED** [review]
LGTM

**Review by petergerlach1 — COMMENTED** [review]
looks good

File: setup.py (hunk 1 above)
Comment (jsture) [reply setup.py:43]: I fixed this for now

**Review by Lauge-Preuss — APPROVED** [review]

**Pull request merged**